### DISCUSSION TITLE
Codelense toggles while editing test case

### DISCUSSION THREAD

**Discussion by Townsheriff** [orta/vscode-jest#772] [Q&A]
Hey,

I'm writing a bunch of unit tests and I'm constantly getting dizzy by codelense appearing over the "it" function call and the disappearing when I start typing and as soon as statement is valid it appears, therefore I could get codelense toggling multiple times per written line. What happens is code moves up and down.

I understand why it happens - code is not parsable, but how do you guys work with that?

Currently I have disabled codelense.

Here is a [loom video](https://www.loom.com/share/d90e4f39abbc4e50915dd30d33b9fc65).

Cheers,
Arturs

**Comment by connectdotz** [comment]
@Townsheriff yeah we are aware that codeLens could interfere with formating/editing. The current thinking is we could probably retire it once we address the "unknown" status issue, see [comment](https://github.com/jest-community/vscode-jest/issues/760#issuecomment-913229066)

**Reply by Townsheriff** [reply]
I see, besides disabling it is there anything I could do? - checkout a specific branch/enable something that prevents codelense to go away/anything else.

I really love the plugin and those buttons make life so beautiful.

**Reply by connectdotz** [reply]
@Townsheriff you can try the prototype `vscode-jest-4.2.0-rc.1.vsix`  from  #773 and let us know if there is any issue. Have fun.
